portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -25,12 +25,13 @@
 import java.time.Instant;
 
 public class Comment {
-    private final String name;
+    private  String name;
```

Comment:
Should be 1 space instead of 2 before String

Reply:
+1